docs/open-source-tizen/tools/mic/mic-reference.md

```diff
@@ -1,274 +1,249 @@
+  --fstab-entry=FSTAB_ENTRY (for raw)
+                      Set fstab entry, 'name' means using device names,
+                      'uuid' means using filesystem uuid
+  ```
 
-- Other options:
+For example, to create a loop image, use the following command:
 
-```
-   --runtime=RUNTIME_MODE
-                       Sets runtime mode, the default is bootstrap mode, valid
-                       values: "native", "bootstrap". "native" means mic uses
-                       localhost environment to create image, while "bootstrap"
-                       means mic uses one tizen chroot environment to create image.
-   --compress-image=COMPRESS_IMAGE (for loop & raw)
-                       Sets the disk image compression. Note: The available
-                       values might depend on the used filesystem type.
-   --compress-disk-image=COMPRESS_IMAGE
-                       Same with --compress-image
-   --shrink (for loop)
-                       Whether to shrink loop images to minimal size
-   --generate-bmap (for raw)
-                       Generate the block map file
-   --fstab-entry=FSTAB_ENTRY (for raw)
-                       Set fstab entry, 'name' means using device names,
-                       'uuid' means using filesystem uuid
+```bash
+$ mic cr loop tizen.ks
 ```
 
-- Examples:
+### Example of Creating a Loop Image
 
-  - create a loop image
+To create a loop image:
 
-    ```bash
-    $ mic cr loop tizen.ks
-    ```
+1. Prepare the kickstart file.
 
-### How to create an image
+   To create an image, you need a proper `.ks` file:
 
-- Prepare kickstart file
+   ```bash
+   # filename: tizen-min.ks
+   lang en_US.UTF-8
+   keyboard us
+   timezone --utc America/Los_Angeles
 
-To create an image, you need a proper ks file.
+   part / --size 1824 --ondisk sda --fstype=ext3
 
-Here's a simple example:
+   rootpw tizen
+   bootloader  --timeout=0  --append="rootdelay=5"
 
-```bash
-# filename: tizen-min.ks
-lang en_US.UTF-8
-keyboard us
-timezone --utc America/Los_Angeles
- 
-part / --size 1824 --ondisk sda --fstype=ext3
- 
-rootpw tizen
-bootloader  --timeout=0  --append="rootdelay=5"
- 
-desktop --autologinuser=tizen
-user --name tizen  --groups audio,video --password 'tizen'
- 
-repo --name=Tizen-base --baseurl=http://download.tizen.org/snapshots/trunk/common/latest/repos/base/ia32/packages/
-repo --name=Tizen-main --baseurl=http://download.tizen.org/snapshots/trunk/common/latest/repos/main/ia32/packages/
- 
-%packages --ignoremissing
-@tizen-bootstrap
-%end
- 
-%post
-rm -rf /var/lib/rpm/__db*
-rpm --rebuilddb
-%end
- 
-%post --nochroot
-%end
-```
+   desktop --autologinuser=tizen
+   user --name tizen  --groups audio,video --password 'tizen'
 
-The ks file above can be used to create a minimum Tizen image. For other repositories, you can replace it with the appropriate repository url.
+   repo --name=Tizen-base --baseurl=http://download.tizen.org/snapshots/trunk/common/latest/repos/base/ia32/packages/
```

Comment:
Link is broken

Reply:
It is legacy. As I mentioned before, mic will be obsoleted. Before then we just leave them.
Samsung doesn't take care of IVI profile.

Reply:
OK